src/scripting/effects.cpp

```diff
@@ -2893,6 +2902,12 @@ uint32_t ef_reduce_pop(EFFECT_PARAMTERS) {
 	ws.world.pop_get_size(trigger::to_pop(primary_slot)) *= amount;
 	return 0;
 }
+uint32_t ef_reduce_pop_abs(EFFECT_PARAMTERS) {
+	auto amount = trigger::read_int32_t_from_payload(tval + 1);
+	assert(std::isfinite(amount));
+	ws.world.pop_get_size(trigger::to_pop(primary_slot)) -= amount;
```

Comment:
this could result in pops with negative size, which could break the game

Reply:
also, asserting that an integer is finite doesn't make too much sense, and is responsible for the CI compilation error